test/libsolidity/syntaxTests/imports/name_clash_imported_free_function.sol

```diff
@@ -0,0 +1,10 @@
+==== Source: s1.sol ====
+function f() pure returns (uint) { return 1337; }
+==== Source: s2.sol ====
+import "s1.sol";
+function f() pure returns (uint) { return 1; }
```

Comment:
This should also definitely not be allowed.
It's not allowed to have ``contract C {}`` in ``s1`` and ``s2`` and this should just behave the same...
Again overloading may be an option, not sure if that was the planned or not or is wise or not, but definitely not identical functions...